src/widget/genericchatitemlayout.cpp

```diff
@@ -99,10 +99,10 @@ QLayout* GenericChatItemLayout::getLayout() const
 int GenericChatItemLayout::indexOfClosestSortedWidget(GenericChatItemWidget* widget) const
 {
     // Binary search: Deferred test of equality.
-    int min = 0, max = layout->count(), mid;
+    int min = 0, max = layout->count();
```

Comment:
Some C++ styles say: "Use narrowest scope as possible, and initialize variables in the declaration"